File: src/main/java/com/my/admin/app/commerce/model/api/response/ProductTypeResponse.java
package com.my.admin.app.commerce.model.api.response;

import com.fasterxml.jackson.annotation.JsonInclude;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

@Data
@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class ProductTypeResponse {

    @ApiModelProperty(value = "Id of product type", example = "333")
    private Long id;

    @ApiModelProperty(value = "Name of product type", example = "Fast food")
    private String name;

    @ApiModelProperty(value = "Description of product type", example = "Fast food")
    private String description;

    @ApiModelProperty(value = "Icon of product type", example = "fast-food")
    private String icon;
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/model/api/response/ProductTypeResponse.kt
package com.bk.reactive.app.commerce.admin.my.model.api.response

import com.fasterxml.jackson.annotation.JsonInclude
import io.swagger.annotations.ApiModelProperty

@JsonInclude(JsonInclude.Include.NON_EMPTY)
class ProductTypeResponse(
    @ApiModelProperty(value = "Id of product type", example = "333")
    val id: String,

    @ApiModelProperty(value = "Name of product type", example = "Fast food")
    val name: String,

    @ApiModelProperty(value = "Description of product type", example = "Fast food")
    val description: String,

    @ApiModelProperty(value = "Icon of product type", example = "fast-food")
    val icon: String,
)